actors/miner/src/lib.rs

```diff
@@ -188,6 +188,10 @@ impl Actor {
             .collect::<Result<_, _>>()?;
 
         let policy = rt.policy();
+        if rt.current_balance() < policy.new_miner_deposit {
+            return Err(actor_error!(illegal_argument, "not enough miner deposit"));
```

Comment:
Insufficient funds is probably a better error code

Reply:
yes, but this may exit two case:
case1: value in message is not enough for new miner deposit 
case2: the from address dont have enough funds to deposit this error return before exec create miner.

so i think argument error should be resonable